frontend/src/lib/components/TimezoneAware/index.tsx

```diff
@@ -36,7 +36,8 @@ function TZConversionHeader(): JSX.Element {
 /** Return a simple label component with timezone conversion UI. */
 export function TZLabel({ time, showSeconds }: { time: string | dayjs.Dayjs; showSeconds?: boolean }): JSX.Element {
     const parsedTime = dayjs.isDayjs(time) ? time : dayjs(time)
-    const { currentTeam } = useValues(teamLogic)
+    const { user } = useValues(userLogic)
+    const currentTeam = user?.team
```

Comment:
Why not `const { currentTeam } = useValues(teamLogic)`?

Reply:
Ahhh because `user` will be loaded much sooner than `currentTeam` (as `currentTeam` brings all the event names, properties, etc.). Because the attribute that we need is included in the `user` response, this can enable the component to be available much sooner. Particularly as it's something that could be loaded quite fast on any scene.